action.php

```diff
@@ -202,8 +202,11 @@ public function handle_loginform(Doku_Event &$event, $param) {
         $form =& $event->data;
         $html = '';
 
-        if ($this->getConf('mailRestriction') !== '') {
-            $html .= sprintf($this->getLang('eMailRestricted'),$this->getConf('mailRestriction'));
+        $validDomains = $hlp->getValidDomains();
+
+        if ($validDomains[0] !== '') {
```

Comment:
no, because explode will always return an array with at least 1 element. However this 1 element could either be an empty string or an domain.